AnnoyancesFilter/MobileApp/sections/mobile-app_specific.txt

```diff
@@ -12,6 +12,9 @@
 ! ##body .smartbanner
 radarbox.com,ciceksepeti.com,rabota.by,temp-mail.org,ortobom.com.br,dicionarioinformal.com.br,5-fifth.com,news.sky.com,chat.babel.com,getwordly.com,school-turnup.com,naked-science.ru,vod.tvp.pl,tvn24.pl,m.lamoda.ru,lkfl2.nalog.ru,kocailem.com,dartworld.shopgate.com,m.alles10euro.de,dodopizza.ru,allo.ua,tempo.co,beymen.com,tvnmeteo.tvn24.pl,outsidetv.com,golfchannel.com,sportsnet.ca,euronews.com,pkobp.pl,tesco.hu,prima.hu,immobilienscout24.de,lcwaikiki.com,m.sigrun-woehr.com,transfermarkt.de,fragfinn.de,hh.ru,eraffe24.de,infranken.de,m.computeruniverse.net,m.emp.de,meduza.global.ssl.fastly.net,meduza.io##body .smartbanner
 !
+navitime.co.jp##.js-banner-frame
+navitime.co.jp##.drive-app-area
+navitime.co.jp##.footer-invite-app-frame
```

Comment:
Please move the rules under the mobile directive.

Reply:
@zloyden Where's exactly?

Reply:
1. Find `!   SECTION: Mobile apps - Mobile sites only` the commented line
2. Move the rules under `townwork.net##div[class^="TM-"][class*="AppBanner"]` it.